ext/intl/locale/locale.stub.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+class Locale
+{
+    /** @return string */
+    public static function getDefault() {}
+
+    /** @return bool */
+    public static function setDefault(string $locale) {}
+
+    /** @return string|null */
+    public static function getPrimaryLanguage(string $locale) {}
+
+    /** @return string|null */
+    public static function getScript(string $locale) {}
+
+    /** @return string|null */
+    public static function getRegion(string $locale) {}
+
+    /** @return array|false|null */
+    public static function getKeywords(string $locale) {}
+
+    /** @return string|false */
+    public static function getDisplayScript(string $locale, ?string $in_locale = null) {}
```

Comment:
```suggestion
    public static function getDisplayScript(string $locale, string $in_locale = UNKNOWN) {}
```

Or make the parameter nullable in the implementation. Same below.

Reply:
I'd prefer the second option, because we'll have an explicit default value at least, and because it makes it possible to pass a second argument even though we want the default behaviour. If you'd prefer otherwise, please comment. :)